package/MDAnalysis/lib/NeighborSearch.py

```diff
@@ -68,7 +68,7 @@ def search(self, atoms, radius, level='A'):
 
         Parameters
         ----------
-        atoms : AtomGroup or Atom
+        atoms : AtomGroup or `Atom`
```

Comment:
But tellling it exactly the class we mean works and it links correctly. That's not bad.